notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -260,7 +260,9 @@
    "source": [
     "### View the Chunks\n",
     "\n",
-    "To view the chunks, run through the following cell. As you can see the document is broken into small pieces with metadata about the chunk based on the document's format"
+    "To view a random sample of the generated chunks, run through the following cell. Re-run the cell to see another random sample. If there are chunking issues, try different conversion and/or chunking configurations.\n",
+    "\n",
+    "As you can see, the document is broken into small pieces with metadata about the chunk based on the document's format."
```

Comment:
You mention the metadata here being in the chunk. It would be nice to have an example or two of chunks within the documentation and point out that metadata. Also we know table chunks can be hard to parse upon first glance. I think showing the user an example just to prepare them for seeing a chunk like that might be helpful

Reply:
I think having an example chunk could either be helpful or confusing here. What if their chunk isn't similar? Is that bad or good?